src/Microsoft.ML.ImageAnalytics/ExtensionsCatalog.cs

```diff
@@ -23,9 +28,12 @@ public static class ImageEstimatorsCatalog
         public static ImageGrayscalingEstimator ConvertToGrayscale(this TransformsCatalog catalog, string outputColumnName, string inputColumnName = null)
             => new ImageGrayscalingEstimator(CatalogUtils.GetEnvironment(catalog), new[] { (outputColumnName, inputColumnName ?? outputColumnName) });
 
-        /// <include file='doc.xml' path='doc/members/member[@name="ImageGrayscalingEstimator"]/*' />
+        ///<summary>
+        /// Create a <see cref="ImageGrayscalingEstimator"/>, which converts images in the column specified in <see cref="InputOutputColumnPair.InputColumnName"/>
```

Comment:
Second method is internal, so main reason why I'm even adding anything here is because I want to get rid of doc.xml.


---
In reply to: [276802631](https://github.com/dotnet/machinelearning/pull/3376#discussion_r276802631) [](ancestors = 276802631)